pandas/tests/frame/test_analytics.py

```diff
@@ -249,6 +249,38 @@ def test_bool_describe_in_mixed_frame(self):
                              index=['count', 'unique', 'top', 'freq'])
         tm.assert_frame_equal(result, expected)
 
+    def test_describe_bool_frame(self):
+        # GH 13891
+        df = pd.DataFrame({
+            'bool_data_1': [False, False, True, True],
+            'bool_data_2': [False, True, True, True]
+        })
+        result = df.describe()
+        expected = DataFrame({'bool_data_1': [4, 2, True, 2],
+                              'bool_data_2': [4, 2, True, 3]},
+                             index=['count', 'unique', 'top', 'freq'])
+        tm.assert_frame_equal(result, expected)
+
+        df = pd.DataFrame({
+            'bool_data': [False, False, True, True, False],
+            'int_data': [0, 1, 2, 3, 4]
+        })
+        result = df.describe()
+        expected = DataFrame({'int_data': [5, 2, df.int_data.std(), 0, 1,
+                                           2, 3, 4]},
+                             index=['count', 'mean', 'std', 'min', '25%',
+                                    '50%', '75%', 'max'])
+        tm.assert_frame_equal(result, expected)
+
+        df = pd.DataFrame({
```

Comment:
i expect both both ``str`` and ``bool`` are described (because these have the same format).